glade_forms/lib/src/validator/int_validator.dart

```diff
@@ -0,0 +1,48 @@
+import 'package:glade_forms/src/src.dart';
+
+class IntValidator extends GladeValidator<int> {
+  IntValidator();
+
+  /// Compares given value with [a] and [b] values.
+  void isBetween({
+    required int a,
```

Comment:
rename to be more clear. Moreover note in doc comment that interval is **inclusive**
a - min
b - max

Reply:
Done, also the interval already has been inclusive, are you suggesting that its not the case? 

Since, the naming is now `min`, `max`, I deleted the case when it checked for b>a.

Reply:
I meant to improve doc comment which explicitly says that interval is inclusive, e.g 

`Given value must be between min and max.`

Reply:
Edited Readme and edited the function `inBetween` with optional param `intervalInclusive`, which specifies if the interval is inclusive or not. By default it's set to true. This is of course also reflected in docs.